packages/common/src/ide/fake/FakeTalonSpokenForms.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  NeedsInitialTalonUpdateError,
+  type SpokenFormEntry,
+  type TalonSpokenForms,
+} from "../../types/TalonSpokenForms";
+import { Notifier } from "../../util/Notifier";
+
+export class FakeTalonSpokenForms implements TalonSpokenForms {
+  private notifier = new Notifier();
+  private entries: SpokenFormEntry[] | null = null;
+
+  async getSpokenFormEntries(): Promise<SpokenFormEntry[]> {
+    if (this.entries == null) {
+      throw new NeedsInitialTalonUpdateError("FakeTalonSpokenForms");
```

Comment:
This is what the file system based implementation does if the json file doesn't exist. The custom spoken form implementation has a special case for this error where it defaults to spoken forms without showing a user error.